### PR TITLE
time kept at kill shelters

### PR DESCRIPTION
**Pull request by ctedtsen**
I added a timeKept attribute to shelters. If a shelter is not a kill shelter, it should be automatically set to "n/a". Otherwise, it must be an integer for the number of days a shelter will keep a pet before they euthanize it. It should be visible on the individual shelter page. I also updated the addShelter form so that is now accounts for this attribute. The seed file also had to be changed to account for it. Run npm run seed then npm start to test it. This attribute was necessary in order to show time left on the kill shelters page :)

### CODE REVIEW

**Comment by ctedtsen** [comment]
I also added the kill shelter page now. There should be a link to in on the shelters page or you can go to the link http://localhost:3000/shelters/kill_shelters. Please do let me know if you have any tips on making it look better cause it's a bit ugly at the moment

File: src/views/shelter/killShelters.handlebars [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<h1>{{title}}</h1>
+
+{{#if shelters.length}}
+    <ul>
+        {{#each shelters}}
+            <li class="tabClass" ><a href="/shelters/{{this._id}}">{{this.shelterName}}</a></li>
+            <li>Pets at this shelter have at most {{this.timeKept}} days until they are euthanized</li>
```
Comment (Jasmine211) [inline src/views/shelter/killShelters.handlebars]: Have all the shelter Information for each one inside a nested <ul> tag. Like this:
`    <ul>
        {{#each shelters}}
            <li class="tabClass" ><a href="/shelters/{{this._id}}">{{this.shelterName}}</a></li>
            <ul>
                <li>Pets at this shelter have at most {{this.timeKept}} days until they are euthanized</li>
                <li>There are currently {{this.numberPets}} pets at this shelter</li>
                {{#if this.pets.length}}  
                    <ul>
                    {{#each this.pets}}
                        <li>{{this.name}}</li>
                    {{/each}}
                    </ul>  
                {{/if}}
            </ul>
        {{/each}}
    </ul>`

**Review by Jasmine211 — CHANGES_REQUESTED** [review]
Looks good, just add some small changes to handlebars page

File: src/views/shelter/killShelters.handlebars (hunk 1 above)
Comment (ctedtsen) [reply src/views/shelter/killShelters.handlebars]: That looks much better! Thanks!

**Comment by ctedtsen** [comment]
I also now added a feature to the shelters page where you can search for shelters in a specific location and it will show a list of them.

File: src/views/shelter/killShelters.handlebars (hunk 1 above)
Comment (Jasmine211) [reply src/views/shelter/killShelters.handlebars]: <ul> 
    {{#each shelters}} 
        <li class="tabClass" ><a href="/shelters/{{this._id}}">{{this.shelterName}}</a> 
        <ul> 
            <li>Pets at this shelter have at most {{this.timeKept}} days until they are euthanized</li> 
            <li>There are currently {{this.numberPets}} pets at this shelter
            {{#if this.pets.length}} 
                <ul> 
                    {{#each this.pets}} 
                        <li>{{this.name}}</li> 
                    {{/each}} 
                </ul> 
            {{/if}} 
            </li>
        </ul> 
        </li>
    {{/each}}
</ul>

**Review by Jasmine211 — APPROVED** [review]
Looks good!

**Pull request merged**